dim-core/src/lib.rs

```diff
@@ -19,7 +19,6 @@
 //! To test run `make test` in the root, or `cargo test` in the root of each module including the
 //! root dir.
 #![allow(opaque_hidden_inferred_bound)]
-#![deny(warnings)]
```

Comment:
Not sure about this.

Reply:
oh yeah, happy to add that back. I removed it out of curiosity to see what it may have been hiding and didn't see anything different.